index.html

```diff
@@ -25,6 +25,41 @@ <h2 class="section-box__title">Products</h2>
       <section class="main-container">
         <section class="section-box section-box--add-product">
           <h2 class="section-box__title">Add product</h2>
+
+          <form class="section-box__item">
+            <div class="section-box__item--wrapper">
+              <label class="section-box__item--title" for="product"
```

Comment:
Któryś raz już pojawia się błąd w nazewnictwach klas, więc jeszcze raz wytłumaczę. 

Załóżmy taki kod:
```
<div class="human">
  <div class="human__head" />
  <div class="human__hand human__hand--left" />
  <div class="human__hand human__hand--right" />
</div>;
```

Piszemy w BEM (Block Element Modifier). Blockiem w naszym przypadku jest "obiekt", który opisujemy, czyli `human`. Elementem większego bloku jest np. część ciała - `human__head` albo `human__hand`. Dany element może mieć modyfikatory, które zmieniają jego wygląd/zachowanie, np. ręka może być lewa albo prawa, więc można zmodyfikować Element: `human__hand` poprzez nadanie dodatkowo klasy `human__hand--left`/`human__hand--right`.

(Nie powinno się zagnieżdzać elementów np. `human__head__hair`, tylko tworzyć nowy element `human__head-hair` albo nowy blok `head`.)

Także po zmianie bazy klas z komentarza wyżej, dostosuj Block, którym jest formularz do nazewnictwa zgodnego z BEMem.

Reply:
na tym przykładzie to jest zrozumiałem, ale jak mam to przenieść na np. formularz to już się troche gubie :D